### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by hezhizhen**
### 1. Why is this pull request needed and what does it do?

Fixed some typos.

### 4. Does this introduce a backward incompatible change or deprecation?

It may be worth noting that `dnsutil.MaximumDefaulTTL` is renamed as `dnsutil.MaximumDefaultTTL`.

### CODE REVIEW

File: plugin/pkg/dnsutil/ttl.go [hunk 1]
```diff
@@ -47,6 +47,6 @@ func MinimalTTL(m *dns.Msg, mt response.Type) time.Duration {
 const (
 	// MinimalDefaultTTL is the absolute lowest TTL we use in CoreDNS.
 	MinimalDefaultTTL = 5 * time.Second
-	// MaximumDefaulTTL is the maximum TTL was use on RRsets in CoreDNS.
-	MaximumDefaulTTL = 1 * time.Hour
+	// MaximumDefaultTTL is the maximum TTL was use on RRsets in CoreDNS.
+	MaximumDefaultTTL = 1 * time.Hour
```
Comment (chrisohaver) [inline plugin/pkg/dnsutil/ttl.go]: This change will be backward incompatible for any external plugins that use these public constants.

Comment (hezhizhen) [reply plugin/pkg/dnsutil/ttl.go]: How about marking as TODO and revert them first?

**Comment by chrisohaver** [comment]
Added to the 1.12 milestone, due to public constant name change: `dnsutil.MaximumDefaulTTL`.

**Comment by hezhizhen** [comment]
> Added to the 1.12 milestone, due to public constant name change: `dnsutil.MaximumDefaulTTL`.

Now that this PR is added to 1.12, should I revert these name changes or not?

**Comment by chrisohaver** [comment]
> > Added to the 1.12 milestone, due to public constant name change: `dnsutil.MaximumDefaulTTL`.
> 
> Now that this PR is added to 1.12, should I revert these name changes or not?

You can, and I will remove it from 1.12 milestone to go out in next patch.

**Comment by hezhizhen** [comment]
> > > Added to the 1.12 milestone, due to public constant name change: `dnsutil.MaximumDefaulTTL`.
> > 
> > 
> > Now that this PR is added to 1.12, should I revert these name changes or not?
> 
> You can, and I will remove it from 1.12 milestone to go out in next patch.

I've reverted this change and added a TODO.

PTAL.

**Review by chrisohaver — APPROVED** [review]

**Pull request merged**